rust/plugin-lib/src/dispatch.rs

```diff
@@ -52,15 +53,55 @@ macro_rules! bail_err {
 /// to the plugin,
 pub struct Dispatcher<'a, P: 'a + Plugin> {
     //TODO: when we add multi-view, this should be an Arc+Mutex/Rc+RefCell
-    views: HashMap<ViewId, View<P::Cache>>,
+    //views: HashMap<ViewId, View<P::Cache>>,
```

Comment:
Please don't leave in commented-out code unless you have a good reason to. If you meant to keep it due to the comment, might as well update the type signature to reflect the one named in the comment.